app/utils/answersHelpers/consignment/houseConsignment/ConsignmentItemAnswersHelper.scala

```diff
@@ -51,6 +56,49 @@ class ConsignmentItemAnswersHelper(
     call = Some(Call(GET, "#"))
   )
 
+  private[houseConsignment] def packageTypeRow(packageIndex: Index): Option[SummaryListRow] = getAnswerAndBuildRow[PackageType](
+    page = PackagingTypePage(houseConsignmentIndex, itemIndex, packageIndex),
+    formatAnswer = formatAsPackage,
+    prefix = "unloadingFindings.rowHeadings.item.packageType",
+    args = Seq(packageIndex.display, itemIndex.display): _*,
+    id = Some(s"change-package-type-${packageIndex.display}"),
+    call = Some(Call(GET, "#"))
+  )
+
+  private[houseConsignment] def packageMarksRow(packageIndex: Index): Option[SummaryListRow] = getAnswerAndBuildRow[String](
+    page = PackagingMarksPage(houseConsignmentIndex, itemIndex, packageIndex),
+    formatAnswer = formatAsText,
+    prefix = "unloadingFindings.rowHeadings.item.packageMarks",
+    args = Seq(packageIndex.display, itemIndex.display): _*,
+    id = Some(s"change-package-mark-${packageIndex.display}"),
+    call = Some(Call(GET, "#"))
+  )
+
+  private[houseConsignment] def packageCountRow(packageIndex: Index): Option[SummaryListRow] = getAnswerAndBuildRow[BigInt](
+    page = PackagingCountPage(houseConsignmentIndex, itemIndex, packageIndex),
+    formatAnswer = formatAsText,
+    prefix = "unloadingFindings.rowHeadings.item.packageCount",
+    args = Seq(packageIndex.display, itemIndex.display): _*,
+    id = Some(s"change-package-count-${packageIndex.display}"),
+    call = Some(Call(GET, "#"))
+  )
```

Comment:
Can these three methods be moved to a package-specific answers helper? Just to keep the structure consistent